projects/vdk-plugins/vdk-logging-format/README.md

```diff
@@ -0,0 +1,68 @@
+This plugins allows for the configuration of the format of VDK logs.
+
+# LTSV
```

Comment:
You should add a few owrlds for "TEXT" . It's a possible value (teh default one) .

Reply:
done